Commons/src/main/java/eu/neclab/ngsildbroker/commons/datatypes/Subscription.java

```diff
@@ -123,6 +133,12 @@ public static Subscription expandSubscription(Map<String, Object> body, String i
 			Object mapValue = mapEntry.getValue();
 
 			switch (key) {
+				case NGSIConstants.NGSI_LD_LOCALONLY:
+					if(!(mapValue instanceof List<?> list && ((Map<String,Object>)(list.get(0))).get("@value") instanceof Boolean)){
```

Comment:
You can take it from NGSIConstants "@value"

Reply:
done